src/algorithms.jl

```diff
@@ -5246,6 +5246,31 @@ E. Hairer, S.P. Norsett, G. Wanner, (1993) Solving Ordinary Differential Equatio
 """
 struct Nystrom4 <: OrdinaryDiffEqPartitionedAlgorithm end
 
+"""
+    FineRKN5()
+
+A 5th order explicit Runge-Kutta-Nyström method which can be applied directly to second order ODEs. 
+Can only be used with fixed time steps.
+
+In case the ODE does not depend on the first derivative, consider using 
+[`Nystrom5VelocityIndependent`](@ref) to increase performance.
```

Comment:
We shouldn't call out this specific method because it's not guaranteed to not change, so this information is non-local. Instead just say that this method requires that the acceleration equation needs to be independent of the velocity.